### ISSUE TITLE
Handling of some version strings produces error messages on RequireNumberSeparators

### ISSUE THREAD

**Issue by jesteves** [Perl-Critic/Perl-Critic#856]
1. **How can it be reproduced**

   ```perl
   my ( $foo, %bar, $baz );

   $foo = v2000;              # error
   %bar = ( v2001 => 1 );     # error
   $baz = $bar{v2002};        # error

   $foo = v20;                # no error
   %bar = ( v21 => 1 );       # no error
   $baz = $bar{v22};          # no error

   $foo = v20.00;             # no error
   %bar = ( v20.01 => 1 );    # no error
   $baz = $bar{v20.02);       # no error

   $foo = 'v2000';            # no error
   %bar = ( 'v2001' => 1 );   # no error
   $baz = $bar{'v2002'};      # no error
   ```

2. **What was expected**

   No error messages on STDERR.

3. **What actually occurred**

   Error messages on STDERR:

   ```console
   Error: Argument "\x{7d0}" isn't numeric in abs at
   ...Perl/Critic/Policy/ValuesAndExpressions/RequireNumberSeparators.pm line 45.
   Error: Argument "\x{7d1}" isn't numeric in abs at
   ...Perl/Critic/Policy/ValuesAndExpressions/RequireNumberSeparators.pm line 45.
   Error: Argument "\x{7d2}" isn't numeric in abs at 
   ...Perl/Critic/Policy/ValuesAndExpressions/RequireNumberSeparators.pm line 45.
   ```

   💡 Observation:

   ```
   Dec  / Hex
   2000 / 7d0 
   2001 / 7d1
   2002 / 7d2
   ```

4. **What version of the involved component**

   * Perl 5.10.1
   * Perl::Critic 1.132;

**Comment by trwyant** [comment]
It looks to me like version numbers (that is, stuff that PPI parses to a PPI::Token::Number::Version) should be exempted from this policy. The syntax is not that of the usual number: multple "decimal points" are allowed, and an underscore is significant.

Exempting PPI::Token::Number::Version is a one-liner (or thereabout) which I am willing to do, but given that this is a PBP policy I think I will give people a week or two to react to this proposal first.

**Comment by petdance** [comment]
This seems reasonable to me.

**Comment by trwyant** [comment]
The above commit has been made into PR #904.

**Comment by petdance** [comment]
This has been fixed with the application of #904.
